setup.py

```diff
@@ -27,7 +27,7 @@ def read(fname):
       author='Parag Baxi',
       author_email='parag.baxi@gmail.com',
       description='QualysGuard(R) Qualys API Package',
-      license ='BSD-new',
+      license="Apache License (2.0)",
```

Comment:
Commented in PR, but `__license__` was already apache, this was a typo by the original author (I assume) to include BSD-new. The problem is on PyPi, this is the field that will get shown. At a minimum, the Apache license file should also be included along with NOTICES and a header. They used `__license__` which you can apply at a package level, but I can add the headers to the files also